localization/ekf_localizer/include/ekf_localizer/string.hpp

```diff
@@ -0,0 +1,29 @@
+// Copyright 2023 Autoware Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef EKF_LOCALIZER__STRING_HPP_
+#define EKF_LOCALIZER__STRING_HPP_
+
+#include <string>
+
+inline std::string eraseBeginSlash(const std::string & s)
```

Comment:
How about `eraseLeadingSlash` or `eraseLeadingForwardSlash`?

Reply:
(Sorry I approved mistakenly, so dismissed it)

Reply:
Updated in [e95c689](https://github.com/autowarefoundation/autoware.universe/pull/2702/commits/e95c68914665df278eda5eddfce8904383789f4b)